kb-api-swagger/src/main/swagger/kb-api.yml

```diff
@@ -85,6 +85,188 @@ paths:
+              summary: Empty filter
+              value: ""
+            oneFilter:
+              summary: Example of a single filter
+              value: "type:text"
+            multipleFilters:
+              summary: Example of a multiple filters
+              value: "type:text  OR author:Ubekendt"
+        - name: sort
+          in: query
+          description: Sort field/direction. The field on which to sort, followed by a space and direction (desc or asc).
+          required: false
+          schema:
+            type: string
+            default: ""
+          allowReserved: true
+          examples:
+            emptySortField:
+              summary: Empty sort field
+              value: ""
+            oneSortField:
+              summary: Example of a single sort field
+              value: "score desc"
+            multipleSortFields:
+              summary: Example of a multiple sort fields
+              value: "score desc, width_pixels asc"
+        - name: start
+          in: query
+          description: Number of leading documents to skip.
+          required: false
+          schema:
+            type: integer
+            default: 0
+          example: 0
+        - name: rows
+          in: query
+          description: Max results per page. This sets the "page size" for paginated search results.
+          required: false
+          schema:
+            type: integer
+            default: 10
+        - name: fl
+          in: query
+          description: Field List. The list of fields to return in the query results.
+          required: false
+          schema:
+            type: string
+            default: ""
+          allowReserved: true
+          examples:
+            emptyField:
+              summary: Empty field
+              value: ""
+            oneField:
+              summary: Example of a single field
+              value: "author"
+            multipleFields:
+              summary: Example of a multiple fields
+              value: "author, title"
+        - name: df
+          in: query
+          description: Default field. Used to configure the q and fq parameters. If not specified, the default field is text
+          required: false
+          schema:
+            type: string
+            default: ""
+        - name: wt
+          in: query
+          description: Response writer.
+          required: false
+          schema:
+            type: string
+            default: "json"
+            enum: [json, xml]
+        - name: facet
+          in: query
+          description: Enable faceting. If set to true, this parameter enables facet counts in the query response. If set to false, this parameter disables faceting. None of the other parameters listed below will have any effect unless this parameter is set to true. The default value is blank (false).
+          required: false
+          schema:
+            type: boolean
+            default: false
+        - name: facet_query
```

Comment:
`facet.query` is a bit advanced. Maybe save it for a later update?

Reply:
Har fjernet fra yml indtil videre